doc/playbook.rst

```diff
@@ -504,11 +507,67 @@ To restore the APDB to a clean state, run ``apdb-cli create-sql`` with ``--drop`
        "postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl" apdb_config_lsstcomcamsim.py
 
 Checking the APDB Version
--------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To identify which schema and ApdbSql version a PostgreSQL APDB is using, run, e.g.:
 
 .. code-block:: sh
 
    psql -h usdf-prompt-processing-dev.slac.stanford.edu lsst-devl rubin \
        -c 'select * from pp_apdb_latiss.metadata;'
+
+Cassandra
+---------
+
+We have a Cassandra cluster at the USDF on dedicated hardware, that is currently deployed in parallell across 12 nodes.
+Of those, 6 are reserved for Andy Salnikov's development and testing, and 6 are available for Prompt Processing.
+The status of the cluster can be monitored with the `cassandra_dashboard`_ in grafana.
+
+.. _cassandra_dashboard: https://grafana.slac.stanford.edu/d/d7d52e6b-e376-49dc-8ef8-e4742dd229a9/cassandra-system-metrics?orgId=1&refresh=5m
+
+The nodes available for Prompt Processing are ``sdfk8sk001`` through ``sdfk8sk006``.
+
+Credentials
+^^^^^^^^^^^
+
+To access the Cassandra, you must add credentials to your ``~/.lsst/db-auth.yaml``.
+The appropriate credentials are stored in the SLAC Vault, under ``rubin/usdf-apdb-dev/cassandra``.
+Add the following to your ``db-auth.yaml``, replacing ``PORT`` and ``PASSWORD`` from the Vault:
+
+.. code-block:: sh
+
+   - url: cassandra://sdfk8sk001:PORT/*
+     username: apdb
+     password: PASSWORD
+   - url: cassandra://sdfk8sk001.sdf.slac.stanford.edu:PORT/*
+     username: apdb
+     password: PASSWORD
+
+Initializing or resetting the APDB
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To set up a new keyspace and connection, use:
+
+.. code-block:: sh
+
+   apdb-cli create-cassandra sdfk8sk001 sdfk8sk004 KEYSPACE APDB_CONFIG.py --replication-factor=3 --enable-replica
+
+Here ``sdfk8sk001`` and ``sdfk8sk004`` are two nodes within the Prompt Processing allocation, which are the ``contact_points`` used for the initial connection.
+All of the available nodes will be used.
+If direct access to the APDB from the RSP is required, note that the ``APDB_CONFIG.py`` will need to be modified by hand to fully specify the hosts:
+
+.. code-block:: python
+
+   config.contact_points=['sdfk8sk001.sdf.slac.stanford.edu', 'sdfk8sk004.sdf.slac.stanford.edu']
```

Comment:
This is incorrect. The config must be set up with `sdfk8sk001.sdf.slac.stanford.edu` in order to be accessible from the Prompt Processing service (why would you want to access it from the RSP?).

Under no circumstances should people be making personal tweaks to the config file; the current architecture is designed on the assumption that there is *one* standardized file used to access any APDB instance. Rather than doing anything clever, 

    apdb-cli create-cassandra sdfk8sk001.sdf.slac.stanford.edu sdfk8sk004.sdf.slac.stanford.edu \
        KEYSPACE APDB_CONFIG.py --replication-factor=3 --enable-replica

creates a file that is guaranteed valid and safe to use.

Reply:
I propose using the conventional keyspace (e.g., `pp_apdb_latiss`) and filename (`pp_apdb_latiss-dev.py`) so the reader doesn't have to hunt for it (the filename in the Postgres example doesn't match what we're actually using, so that should be fixed).

Reply:
That's interesting, I didn't realize `sdfk8sk001.sdf.slac.stanford.edu` would be necessary for Prompt Processing. The example I gave worked fine in my batch mode testing. I will remove the reference to the RSP and update the examples.